pytest/tests/sanity/upgradable.py

```diff
@@ -341,68 +270,45 @@ def upgrade_nodes(self) -> None:
             )
 
     def wait_epoch(self) -> None:
+        """Wait until the next epoch starts."""
         start_epoch = self._rpc_node.get_epoch_id()
         logger.info(f"Current epoch is {start_epoch}. Waiting for next epoch")
+        condition = lambda h: self._rpc_node.get_epoch_id(block_height=h
+                                                         ) != start_epoch
```

Comment:
nit: the formatting is a bit wild here, can you put it in brackets or sth to make it better? or helper method